lib/Runtime/PlatformAgnostic/Platform/Linux/PerfTrace.cpp

```diff
@@ -84,13 +84,19 @@ void  PerfTrace::WritePerfMap()
 
             scriptContext->MapFunction([=] (FunctionBody* body)
             {
+              char16 const * url = body->GetSourceContextInfo()->url;
+              if (body->GetSourceContextInfo()->IsDynamic()) {
+                url = _u("eval");
```

Comment:
nit: i'd change this to dyn or something like that since strictly speaking we can have other dynamic code (eg. `new Function("...")`)